srttools/tests/test_calibration.py

```diff
@@ -30,7 +30,7 @@ def logger():
     return logger
 
 
-np.random.seed(1241347)
+np.random.seed(124137)
```

Comment:
Tip: you can initialize the random seed by providing the current time

Reply:
I'm using always the same to be consistent between tests ran at different times